src/main/scala/org/deepdive/datastore/Dataloader.scala

```diff
@@ -26,8 +26,10 @@ class DataLoader extends JdbcDataStore with Logging {
    * @param dbSettings: database settings (DD's class)
    * @param usingGreenplum: whether to use greenplum's gpunload
    * @param query: the query to be dumped
+   * @param folder: for greenplum, the relative path to gpfdist
    */
-  def unload(filename: String, filepath: String, dbSettings: DbSettings, usingGreenplum: Boolean, query: String) : Unit = {
+  def unload(filename: String, filepath: String, dbSettings: DbSettings, 
+    usingGreenplum: Boolean, query: String, folder: String) : Unit = {
```

Comment:
It can be an arbitrary subdirectory structure. The folder argument is managed by DeepDive, and won't be exposed to users. If "folder" is invalid, "executeSqlQueries" will throw an exception.